src/components/access-type.js

```diff
@@ -163,22 +166,22 @@ class AccessTypeBase extends React.Component {
     return [];
   };
 
-  runSequentialCalls = async (storyId = "") => {
+  runSequentialCalls = async (callback = () => null) => {
```

Comment:
`storyId` will be undefined always as we're calling `runSequentialCalls` with empty args. I don't know how this came here though. @shreyaa1 @Deokumarjnu